data/helpers.d/backend

```diff
@@ -51,3 +51,41 @@ ynh_remove_logrotate () {
 		sudo rm "/etc/logrotate.d/$app"
 	fi
 }
+
+# Create a dedicated nginx config
+#
+# usage: ynh_add_nginx_config
+ynh_add_nginx_config () {
+	finalnginxconf="/etc/nginx/conf.d/$domain.d/$app.conf"
+	ynh_compare_checksum_config "$finalnginxconf" 1
+	sudo cp ../conf/nginx.conf "$finalnginxconf"
```

Comment:
Could it be possible to pass the "source" filename as a parameter?
For example I have several apps for which I had to come up with 2 nginx conf files depending of whether the app is install at root or on a subdir (for example [here](https://github.com/YunoHost-Apps/wallabag2_ynh/tree/code_refactoring/conf)).